src/analytics/Analytics.ts

```diff
@@ -231,12 +259,41 @@ export class Analytics {
         });
     }
 
+    /**
+     * Tracking for download file action V2
+     *
+     * @param {SourceType} sourceType
+     * @param {number} contentItemId
+     * @param {number} itemId
+     */
+    trackDownloadFileV2(
```

Comment:
i've changed the download url path to: 
/live_board/v2/content_items/${contentItemId}/downloads
instead of
/live_board/v2/downloads